src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -345,14 +346,18 @@ void GNAPlugin::Init() {
 
 void GNAPlugin::InitGNADevice() {
     OV_ITT_SCOPED_TASK(itt::domains::GNA_LT, "InitGNADevice");
-    gnadevice = std::make_shared<GNADeviceHelper>(config.gnaExecTarget,
-                config.gnaCompileTarget,
-                config.swExactMode,
-                gnaFlags->performance_counting,
-                !config.dumpXNNPath.empty(),
-                GetDeviceVersionFromString(config.dumpXNNGeneration));
-    size_t page_size_bytes = 4096;
-    gnamem = std::make_shared<gna_memory_type>(memory::make_polymorph<memory::GNAAllocator>(gnadevice), page_size_bytes);
+    if (gnaFlags->sw_fp32) {
+        gnamem.reset(new gna_memory_type(memory::make_polymorph<std::allocator<uint8_t>>()));
+    } else {
+        gnadevice = std::make_shared<GNADeviceHelper>(config.gnaExecTarget,
+                    config.gnaCompileTarget,
```

Comment:
Config can be defined during load network

Reply:
it still can be defined and will overwrite the default configuration(core.setProperty(conf))